src/leases/components/leaseSections/rent/BasisOfRent.js

```diff
@@ -444,10 +427,10 @@ const BasisOfRent = ({
               isFieldAllowedToRead(leaseAttributes, LeaseBasisOfRentsFieldPaths.AMOUNT_PER_AREA)
             }>
               {calculatorType === CalculatorTypes.TEMPORARY && <FormTextTitle enableUiDataEdit uiDataKey={getUiDataLeaseKey(LeaseBasisOfRentsFieldPaths.BASE_YEAR_RENT)}>
-                {'Vuokra/kuukausi'}
+                {'Vuokra/kk'}
               </FormTextTitle>}
               {calculatorType !== CalculatorTypes.TEMPORARY && <FormTextTitle enableUiDataEdit uiDataKey={getUiDataLeaseKey(LeaseBasisOfRentsFieldPaths.BASE_YEAR_RENT)}>
-                {LeaseBasisOfRentsFieldTitles.RENT}
+                {'Vuokra/vuosi'}
```

Comment:
I kinda feel that these titles need a bit of standardisation. Either all of them in a component should be defined in the title enums (which has been historically the default for the project as far as I know), or all of them should be written here directly, but not a mix of those two. (Which was already the case prior to this PR I guess.)

Reply:
There has been a mix of these two. I'll improve these a bit and convert these to ENUMS